src/Core/Internal/DefaultGraphQLExecuter.cs

```diff
@@ -65,6 +65,9 @@ protected virtual ExecutionOptions GetOptions(string operationName, string query
                 .Concat(DocumentValidator.CoreRules())
                 .ToList();
 
+            // add Schema FieldNameConverter to overrride default CamelCaseFieldNameConverter
+            opts.FieldNameConverter = Schema.FieldNameConverter;
```

Comment:
I suggest putting this in the initializer where the other options are set, at line 57.

```c#
var opts = new ExecutionOptions()
{
	// ...
	FieldNameConverter = Schema.FieldNameConverter
}
```

Reply:
I changed it.